README.es.md

```diff
@@ -0,0 +1,242 @@
+|[workers](https://docs.onfleet.com/reference/workers)|get(id=nil, queryParameters={})<br />get_tasks(id)<br />get_by_location(longitude, latitude, radius)<br />get_schedule(id)|create(body={})<br />set_schedule(id, body={})<br />match_metadata(body={})|update(id, body={})<br />insert_task(id, body={})|delete(id)|
+
+### **Peticiones GET**
+Para obtener todos los objetos de entidad dentro de un punto final, use `list`:
+
+```
+list()
+```
+
+Examples of `list()`:
+
+```
+tasks = Onfleet::Tasks.new
+tasks.list(config)
+tasks.list(config, queryParameters{})
+```
+
+Opcionalmente, puede enviar un hash de parámetros de consulta para ciertos puntos finales. El hash de Ruby se codificará en parámetros de consulta de URL utilizando la gema `uri`. Referirse de nuevo a [API documentation](https://docs.onfleet.com/) para puntos finales que admiten parámetros de consulta.
+
+```
+tasks = Onfleet::Tasks.new
+tasks.list(config, queryParameters={'from': '1455072025000', 'state': '1, 2, 3'})
+```
+
+Para obtener un objeto de entidad dentro de un punto final, especifique un `entity id`:
+
+```
+# obtener ejemplos con la búsqueda de entityId
+tasks = Onfleet::Tasks.new
+tasks.get(config, 'taskId')
+
+recipients = Onfleet::Recipients.new
+recipients.get(config, 'workerId')
+```
+
+Junto con la búsqueda de un objeto de entidad con un `id` asociado, los siguientes parámetros de consulta también están disponibles en un grupo selecto de puntos finales:
+
+* `queryParameters` (hash)
+* `name`
+* `entity`
+* `phone`
+* `shortId`
+
+```
+# obtener ejemplos con argumentos adicionales
+workers = Onfleet::Workers.new
+workers.get(config, 'workerId', queryParameters={'analytics': 'true'})
+
+containers = Onfleet::Containers.new
+containers.get(config, 'workers', 'workerId')
+containers.get(config, 'teams', 'teamId')
+containers.get(config, 'organizations', 'organizationId')
+```
+
+Para obtener un controlador por ubicación, use el método `get_by_location`:
+```
+workers = Onfleet::Workers.new
+worker.get_by_location(config, 'longitude_value', 'latitude_value', 'radius_value')
+```
+El valor predeterminado del `radius` es 1000 metros si no se proporciona como argumento.
+
+### **Peticiones POST**
+Para crear un objeto de entidad dentro de un punto final:
+```
+.create(config, body={})
+```
+
+Ejemplos de `create()`:
+```
+body = {
+  "name": "A Swartz",
+  "phone": "617-342-8853",
+  "teams": [
+    "nz1nG1Hpx9EHjQCJsT2VAs~o"
+  ],
+  "vehicle": {
+    "type": "CAR",
+    "description": "Tesla Model 3",
+    "licensePlate": "FKNS9A",
+    "color": "purple"
+  }
+}
+
+workers = Onfleet::Workers.new
+workers.create(config, body)
+```
+
+Las solicitudes POST extendidas incluyen clon, batch_create, auto_assign en el punto final de las tareas; set_schedule en el punto final de los trabajadores; y auto_dispatch en el punto final de los equipos. A continuación, se muestran ejemplos de estos puntos finales:
```

Comment:
Let's change `punto final` to `endpoint` in the whole document  since punto final sounds weird xD
Also, `puntos finales` to `endpoints`.